daemon/Makefile

```diff
@@ -19,7 +19,9 @@ install: all
 .PHONY: check-bindata
 check-bindata:
 	@which go-bindata > /dev/null 2> /dev/null || exit 0 && \
-	(go-bindata -prefix ../ -ignore Makefile -ignore '.+\.o$$' -o bindata.tmp ../bpf/...; \
+	(go-bindata -prefix ../ -mode 0640 -ignore Makefile -ignore bpf_features.h \
```

Comment:
We should store the gobindata command in a variable and use it for both and just add the `-o` for one of them.